packages/frontend/src/components/EmojiGrid.tsx

```diff
@@ -28,14 +30,15 @@ const EmojiGrid: React.FC<EmojiGridProps> = ({ topEmojis, socket, lang }) => {
           const rowCount = Math.ceil(topEmojis.length / columnCount);
 
           return (
-            <Grid
+            <Grid<EmojiGridItemData>
```

Comment:
mildly cursed syntax of passing a generic type to a JSX element
https://mariusschulz.com/blog/passing-generics-to-jsx-elements-in-typescript

Reply:
oh wow.